blocks/header/header.js

```diff
@@ -1,8 +1,9 @@
 import { getMetadata } from '../../scripts/aem.js';
+import { customDecoreateIcons } from '../../scripts/scripts.js';
 import { loadFragment } from '../fragment/fragment.js';
 
 // media query match that indicates mobile/tablet width
-const isDesktop = window.matchMedia('(min-width: 992px)');
+const isDesktop = window.matchMedia('(min-width: 1024px)');
```

Comment:
i think this is 1025 as per https://ncchat.slack.com/archives/C07KG702WMC/p1726052767589049